Lucidtech/Las/Client.cs

```diff
@@ -441,7 +501,7 @@ public object CreateBatch(string description)
             string? sortBy = null,
             string? order = null
         ) {
-            var queryParams = new Dictionary<string, object> {
+            var queryParams = new Dictionary<string, object?> {
                 {"status", statuses},
                 {"executionId", executionIds},
             };
```

Comment:
status and executionId should not be added unless they are provided (and is a string) by the user